test/upgrade/upgrade_test.sh

```diff
@@ -13,65 +13,100 @@
+
+  accumulo org.apache.accumulo.test.TestIngest -i uno -u root -p secret --timestamp 1 --size 50 --random 56 --rows 200000 --start 0 --cols 1  --createTable --splits 10
+
+  accumulo shell -u root -p secret <<EOF
+   table test_ingest
+   importdirectory $BULK/bulk $BULK/fail false
+   createtable foo
+   config -t foo -s table.compaction.major.ratio=2
+   insert r1 f1 q1 v1
+   flush -t foo -w
+   scan -t accumulo.metadata -c file
+   insert r1 f1 q2 v2
+   insert r2 f1 q1 v3
+EOF
+)
 
-"$PREV/bin/accumulo" init --clear-instance-name --instance-name $INSTANCE --password secret
-"$PREV/bin/start-all.sh"
-
-"$PREV/bin/accumulo" org.apache.accumulo.test.TestIngest -i $INSTANCE -u root -p secret --timestamp 1 --size 50 --random 56 --rows 200000 --start 0 --cols 1  --createTable --splits 10
-"$PREV/bin/accumulo" org.apache.accumulo.test.TestIngest -i $INSTANCE -u root -p secret --rfile $BULK/bulk/test --timestamp 1 --size 50 --random 56 --rows 200000 --start 200000 --cols 1
-
-echo -e "table test_ingest\nimportdirectory $BULK/bulk $BULK/fail false" | $PREV/bin/accumulo shell -u root -p secret
 if [[ $1 == dirty ]]; then
-	pkill -9 -f accumulo.start
-else 
-	"$PREV/bin/stop-all.sh"
+	pkill -9 -f accumulo\\.start
+else
+  (
+    eval "$(./bin/uno env)"
+    "$ACCUMULO_HOME/bin/stop-all.sh"
+  )
 fi
 
-echo "==== Starting Current ==="
-
-"$CURR/bin/accumulo-cluster" start
-"$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 1 --random 56 --rows 400000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-echo "compact -t test_ingest -w" | $CURR/bin/accumulo shell -u root -p secret
-"$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 1 --random 56 --rows 400000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-
-
-"$CURR/bin/accumulo" org.apache.accumulo.test.TestIngest --timestamp 2 --size 50 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-"$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 2 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-echo "compact -t test_ingest -w" | $CURR/bin/accumulo shell -u root -p secret
-"$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 2 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-
-"$CURR/bin/accumulo-cluster" stop
-"$CURR/bin/accumulo-cluster" start
-
-"$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 2 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-
-pkill -9 -f accumulo.start
-"$CURR/bin/accumulo-cluster" start
-
-"$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 2 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
-
+cd $ACCUMULO_DIR
+git checkout master
+git clean -xfd
+cd $UNO_DIR
+./bin/uno fetch accumulo
+./bin/uno install accumulo --no-deps
+./install/accumulo*/bin/accumulo-cluster start
+(
```

Comment:
pretty sure this subshell is unnecessary

Reply:
yeah, at this point in the script I was just doing it for consistency